### PR TITLE
Release v0.0.9-BETA

### PR DESCRIPTION
**Pull request by xRodzMC**
**This is an updated version of dev-commit. These needs to be tested in PocketMine-MP 5.0.0 in the future.**

Dylan (dktapps) has announcement about the PocketMine-MP 5.0.0 which supports many advantages of 4.0.0, this release will be able to use to test them to 5.0.0. When its done and ready to publish, we will publish the v0.1.0 which supports 5.0.0 version of PocketMine-MP.

*If you have a question about this pull request, you can comment down below or review this commit.*

### CODE REVIEW

File: plugin.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
 name: ProfanityFilter
-version: 0.0.8
+version: 0.0.9-BETA
 main: ReinfyTeam\ProfanityFilter\Loader
-api: 4.0.0
+api: [4.0.0,5.0.0]
```
Comment (NhanAZ) [inline plugin.yml:5]: Does Poggit allows this?
4.0.0 and 5.0.0 at the same time

Comment (xqwtxon) [reply plugin.yml:5]: I'll ask later...

Comment (NhanAZ) [inline plugin.yml:5]:
```suggestion
api: [5.0.0]
```

**Comment by xqwtxon** [comment]
Since poggit was unable to use `src-namespace-prefix` the build will returning warning of namespace.

**Review by NhanAZ — APPROVED** [review]

**Pull request closed without merging**